run_dir/design/workset_samples.html

```diff
@@ -13,27 +13,78 @@
 <div id="page_content" style="display:none;">
 
     <h1>Workset <span id="page_title">{{ workset_name}}</span></h1>
-    <button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#modal_plate">
+    <h3>Lims Step : <span id="span_lims_step"></span>, run on <span id="date_run"></span></h3>
+    <div id="modal_button">
+        <button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#modal_plate">
           Launch plate view
-      </button>
-    <div id="workset_tables">
+        </button>
     </div>
-    <div id="modal_plate" class="modal fade" >
-        <div class="plate_modal modal-dialog"> 
-             <div class="modal-content">
+    <div class="tabbable">
+        <ul class="nav nav-tabs">
+        <li class="active"><a href="#workset_tables" role="tab" data-toggle="tab">Workset Tables</a></li>
+        <li><a href="#workset_notes_panel" role="tab" data-toggle="tab">Workset Notes</a></li>
+        </ul>
+        <div class="tab-content">
+        <div class="tab-pane fade in active" id="workset_tables">
+        </div> 
+        <div class="tab-pane fade" id="workset_notes_panel">
+          <form action="" method="POST" id="workset_notes_form" role="form">
+            <div class="form-group">
+              <label for="new_note_text">Insert new running note:</label>
+              <textarea rows="5" class="form-control" id="new_note_text"></textarea>
+            </div>
+            <button type="submit" class="btn btn-primary" id="save_note_button">Preview workset  note</button>
+          </form>
+          <div class="panel-group" id="workset_notes"></div>
+        </div>
+        </div>
+        <div id="modal_plate" class="modal fade" >
+            <div class="plate_modal modal-dialog"> 
+                 <div class="modal-content">
                 <input type="button" Onclick="updatePlate('rec_ctrl')" value=" Reception Control " />
                 <input type="button" Onclick="updatePlate('library')" value=" Library Validation " />
                 <input type="button" Onclick="updatePlate('seq')" value=" Sequencing" />
                 <canvas id="myCanvas" width="1278" height="860" style="border:1px solid #000000;">
```

Comment:
Fair enough - I just couldn't find `myCanvas` referred to anywhere else in the code so I was confused..